src/lang/en.json

```
@@ -1024,6 +1024,17 @@
     "useRemoteBrowser": "Use a Remote Browser",
     "deleteRemoteBrowserMessage": "Are you sure want to delete this Remote Browser for all monitors?",
     "GrafanaOncallUrl": "Grafana Oncall URL",
+    "systemService": "System Service",
+    "systemServiceName": "Service Name",
+    "systemServiceDescription": "Checks if system service",
+    "systemServiceDescriptionLinux": "Checks if Linux systemd service",
+    "systemServiceDescriptionWindows": "Checks if Windows Service Manager",
+    "systemServiceActiveText": "service is active.",
+    "systemServiceLinuxLabel": "Show Linux Debug Command",
+    "systemServiceWindowsLabel": "Show Windows Debug Command",
+    "systemServiceExpectedOutput": "Expected Output",
+    "systemServiceActiveResultLinux": "\"active\"",
+    "systemServiceActiveResultWindows": "\"Running\"",
```

this way we remove untranslatable things and are more translatable for RTL languages

```suggestion
    "systemServiceExpectedOutput": "Expected Output: \"{0}\"",
```